### PR TITLE
Suggested changes for SC member selection

### PR DESCRIPTION
**Pull request by vmbrasseur**
Per conversations in the October SC call and the SC Slack channel.

This PR is to get conversation going and to iterate as necessary.

* Specify appointed member terms start Feb 1 & last 2 years
* Give more wiggle room in the nomination/election timing (Sept-Nov
instead of Sept only)
• Specify appointments happen at start of year after newly elected
member terms begin

### CODE REVIEW

**Comment by kpfleming** [comment]
Looks good to me, thanks for doing this!

File: CHARTER.adoc
Comment (DuaneOBrien) [inline CHARTER.adoc:27]: "Typically an appointed term will be February 1st for term start and January 31st two years later for term end."

This sentence sticks out to me a little, as it's language that serves as guidance, but isn't binding or prescriptive the way the rest of this section is. If it's included as an example, I think we should clarify the language a bit to say it's an example. If it's included to drive the SC to make a decision and not drag their feet, I think we should clarify the language in a way that sets it as a clear expectation of the SC. 

While I favor a clarification to this piece of language, I don't think it's enough of an issue to hold up the proposed changes.

Comment (vmbrasseur) [reply CHARTER.adoc]: Fixed in the next commit.

**Comment by DuaneOBrien** [comment]
I'm +1 for merging these changes to the charter.

**Comment by ashleywolf** [comment]
+1 from me.

File: resolutions/numbers.md
Comment (DuaneOBrien) [inline resolutions/numbers.md:7]: This file wasn't updated to capture resolutions 3 and 4, at least one of which was mine (Mea Culpa). This one should be resolution 5.

Resolutions 3 and 4 are in the approved folder.

File: resolutions/proposed/2020-03.md
Comment (DuaneOBrien) [inline resolutions/proposed/2020-03.md:2]: We'll need to change the number here as well and rename the file.

Comment (vmbrasseur) [reply resolutions/proposed/2020-03.md]: Done.

File: resolutions/numbers.md
Comment (vmbrasseur) [reply resolutions/numbers.md]: All fixed. :)

**Comment by DuaneOBrien** [comment]
+1 to merge, my feedback was addressed

**Review by ashleywolf — APPROVED** [review]
+1

**Comment by kpfleming** [comment]
Since we've all already voted to make these changes the charter, the resolution embedded in this PR should be 'approved' and not 'proposed'. @vmbrasseur do you want me to update the branch to relfect that?

**Comment by vmbrasseur** [comment]
@kpfleming I've become confused by the order of operations on this one. The resolution will need updating to reflect approval date, votes, etc. along with being moved to approved. Is the idea to do all that—updating the PR—and then merging it?

**Comment by vmbrasseur** [comment]
In that case we have a minor and quickly fixable problem. I'm working on filling in the vote results and the most approves I can puzzle out is four:

* Me
* KPF
* Duane
* Ashley

I can't find any record of +1/-1 from @decause, @jeffmcaffer, or @caniszczyk. Changes to the charter require ⅔ majority, so we need one more person to sign off on this.

In the meantime, I'll send over a mostly-complete update to this PR. Once we get one more +1, I can update just that line in the resolution and we can merge this.

**Comment by geekygirldawn** [comment]
@vmbrasseur @kpfleming This has been sitting around a while - is it ready to be merged?

**Comment by justaugustus** [comment]
> @vmbrasseur @kpfleming This has been sitting around a while - is it ready to be merged?

Merge conflict with https://github.com/todogroup/governance/pull/104.
https://github.com/todogroup/governance/pull/158 should help with that.

**Comment by kpfleming** [comment]
This will have to taken up by the 2022 SC to get it completed.

**Comment by vmbrasseur** [comment]
Summary: This was only "approved", not APPROVED. One more vote was needed and that didn't happen before the end of my 2020 term.

The entire SC has changed since then, so I suspect this will need to start from scratch. We can talk about it on tomorrow's call if there's time. (I've added it to the agenda)

**Comment by anajsana** [comment]
I think this is related to this new issue: https://github.com/todogroup/governance/issues/150

**Comment by anajsana** [comment]
> I think this is related to this new issue: #150

We will be using this issue now

**Pull request closed without merging**
